### ISSUE TITLE
Dota 2 or Steam gifts

### ISSUE THREAD

**Issue by LionH3art** [jshackles/idle_master#214]
does Idle master works for gaining dota 2 items or steam gifts?

**Comment by Tsunamical** [comment]
No, it idles for Steam trading cards only. It essentially simulates playing the game by setting your status to playing it. To get Dota items you have to play the game. There aren't any other Steam items you can get as a drop other than cards.

**Comment by LionH3art** [comment]
so how can i get a drop?
i just have dota . . .

**Comment by Wyn10** [comment]
Buy more games.

**Comment by LionH3art** [comment]
so tell me some of those games names.

**Comment by Tsunamical** [comment]
Any game that has says it has trading cards on the store page. This issue should be closed now as this is not a problem with the program. Issues should only be used for issues with the program. All the information you need to use this properly is there for you to read, or look at the FAQ, which you can find if you read.
